opm/parser/eclipse/EclipseState/Schedule/DynamicState.hpp

```diff
@@ -68,6 +68,17 @@ namespace Opm {
         }
 
 
+        const T& back() const {
+            if (m_data.size() > 0)
+                return m_data.back();
+            else
+                return m_initialValue;
+        }
+
+        T& back() {
```

Comment:
Is this necessary. I am slighly uncomfortable to let this be a mutable return, since modifying the initialvalue semantically feels different from modifying an arbitrary last value.

Maybe it could throw if size == 0?